server/embed/etcd.go

```diff
@@ -155,6 +155,15 @@ func StartEtcd(inCfg *Config) (e *Etcd, err error) {
 		}
 	}
 
+	// OpenBackendTimeout defaults to "0" if not set.
```

Comment:
This comment is misleading as it defaults to empty string (if we consider initial value) 
or it defaults to "10s"  if we consider the produced value. The comments below are talking about the 'resolved' value.